<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importResultsData</class>
 <widget class="QDialog" name="importResultsData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>225</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import Results File </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Total Head (H)              </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="totalHeadLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse1">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X-displacement (U)        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xDispLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse2">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y-displacement (V)        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yDispLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse3">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Z-displacement (W)       </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="zDispLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse4">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Skip Heading (H,U,V,W)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="skipHeading1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="skipHeading2">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="skipHeading3">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="skipHeading4">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Skip heading is the number of rows which is skipped when impporing ASCII file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Start Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
